src/main/resources/standardTestCases/globbingBehavior.test

```diff
@@ -0,0 +1,12 @@
+name: globbingBehavior
+testFormat: workflowsuccess
+
+files {
+  wdl: globbingBehavior/globbingBehavior.wdl
+}
+
+metadata {
+  workflowName: globbingBehavior
+  status: Succeeded
+  "outputs.globbingBehavior_countArrays_out": "[2, 2, 3, 2]"
```

Comment:
ToL (as below) if this is possible to make into a real validation, that'd be awesome...